### PR TITLE
Allow for the ability to set Pushover notification sound and priority…

### PR DESCRIPTION
**Pull request by pooley182**
… from within Ombi.

### CODE REVIEW

File: src/Ombi.Settings/Settings/Models/Notifications/PushoverSettings.cs [hunk 1]
```diff
@@ -8,5 +8,7 @@ public class PushoverSettings : Settings
         public bool Enabled { get; set; }
         public string AccessToken { get; set; }
         public string UserToken { get; set; }
+        public sbyte Priority { get; set; }
```
[the remark this answers is not in the document]
Comment (pooley182) [reply src/Ombi.Settings/Settings/Models/Notifications/PushoverSettings.cs]: the possible options for priority are only -2, -1, 0, 1. Didn't really need the full range of an int. We can change it if you'd prefer though.

File: src/Ombi.Notifications/Agents/PushoverNotification.cs [hunk 2]
```diff
@@ -177,7 +177,7 @@ protected override async Task Send(NotificationMessage model, PushoverSettings s
         {
             try
             {
-                await Api.PushAsync(settings.AccessToken, model.Message, settings.UserToken);
+                await Api.PushAsync(settings.AccessToken, model.Message, settings.UserToken, settings.Priority, settings.Sound);
```
[the remark this answers is not in the document]
Comment (tidusjar) [reply src/Ombi.Notifications/Agents/PushoverNotification.cs]: Yeah that should be fine regarding priority.

I'd just do `settings.Sound.HasValue() ? settings.Sound : "pushover"` assuming that `pushover` is the default sound?

[the remark this answers is not in the document]
Comment (tidusjar) [reply src/Ombi.Notifications/Agents/PushoverNotification.cs]: Or do what you have done, that's fine!

File: src/Ombi.Settings/Settings/Models/Notifications/PushoverSettings.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (tidusjar) [reply src/Ombi.Settings/Settings/Models/Notifications/PushoverSettings.cs]: Ok that's fine, as long as you can justify the reason :)

**Comment by tidusjar** [comment]
Thanks!

**Pull request merged**